openassessment/xblock/openassessmentblock.py

```diff
@@ -809,16 +810,17 @@ def get_waiting_details(self, status_details):
             >>> self.get_waiting_details(status_details)
             "peer"
         """
-        waiting = None
-        peer_waiting = "peer" in status_details and not status_details["peer"]["graded"]
-        ai_waiting = "ai" in status_details and not status_details["ai"]["graded"]
-        if peer_waiting and ai_waiting:
-            waiting = "all"
-        elif peer_waiting:
-            waiting = "peer"
-        elif ai_waiting:
-            waiting = "example-based"
-        return waiting
+        waiting = []
+        modules = [('peer', 'peer'), ('ai', 'example-based'), ('staff', 'staff')]
+        for module in modules:
+            if module[0] in status_details and not status_details[module[0]]['graded']:
+                waiting.append(module[1])
+        num_details = len(waiting)
+        if num_details == 0:
+            return None
+        elif num_details == len(modules):
+            return "all"
+        return ','.join(waiting)
```

Comment:
This block is code I wrote in the previous PR. If we're no longer differentiating between the different waiting states in the html template, do we really need to do all of this, or just return a bool indicating whether we're in a waiting state?